package.json

```diff
@@ -20,7 +20,7 @@
     "chokidar": "^2.1.1",
     "daemonize2": "^0.4.2",
     "event-stream": "^4.0.1",
-    "iobroker.objects-redis": "^0.2.7",
+    "iobroker.objects-redis": "^0.3.0",
```

Comment:
Do we want to adopt the `@iobroker/xyz` naming here?

Reply:
Will clarify with BF